src/github/membership/index.ts

```diff
@@ -0,0 +1,151 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import { repository as getRepository, token as getToken } from '../index.js';
+import { handleErrorMessage, handleSuccessMessage } from '../../utils/index.js';
+import { Command } from 'commander';
+import { Octokit } from '@octokit/rest';
+import { Ora } from 'ora';
+
+interface IGitHubMembershipArgs {
+  org?: string;
+  users?: string[];
```

Comment:
```suggestion
  users?: string | string[];
```

Reply:
Nah. For TS API usage, I think we want to force the developer to give us a `string[]`